### ISSUE TITLE
encoder binary classification tasks

### ISSUE THREAD

**Issue by ismael-mendoza** [prob-ml/bliss#28]
Hi @jeff-regier, here is a proposed task list for the new PR related to fixing the new combined encoder so that it can detect galaxies, 

- [ ] Issues in #46 

- [x] Fix any remaining bugs in `sourcenet.py` and `sleep.py` (currently predicts stars with 100% probability) 

- [ ] Add a 1/3 galaxies sleep phase test and pass it. 

- [ ] Add a one star & one galaxy test and pass it. 

- [ ] Figure out how to train the encoder **only once** but still pass all the above tests and the existing star one too.

Here is also a list of questions and roadblocks that I can see along the way: 

- [x] I remember that we used different padding configurations for stars and galaxies (galaxies are bigger than stars). If we want to classify and find both, do we need to find a "sweetspot" ? 

- [x] Is training the encoder only once (see above) too ambitious? 

Please let me know if you have additional suggestions/tasks. Thanks!

**Comment by ismael-mendoza** [comment]
This issue is not useful, I prefer the smaller issues like #123 , so I will close it and reopen a more specific one if appropriate later.
